### PR TITLE
Updating the tables

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/25058

### CODE REVIEW

File: governance/etcd_encryption_policy.adoc [hunk 1]
```diff
@@ -47,48 +47,52 @@ spec:
...
 | metadata.name
-| Required.
-The name for identifying the policy resource.
+| Required
+| The name for identifying the policy resource.
 
 | metadata.namespaces
-| Optional.
+| Optional
+|
```
Comment (dockerymick) [inline governance/etcd_encryption_policy.adoc]: @dhaiducek can you help provide a description here. I'll apply your suggestion to the other files.

File: governance/config_policy_ctrl.adoc [hunk 2]
```diff
@@ -98,36 +98,42 @@ spec:
...
-| Required for namespaced objects or resources. The namespaces in the hub cluster that the policy is applied to.
+| Required for namespaced objects or resources.
+| The namespaces in the hub cluster that the policy is applied to.
 Enter at least one namespace for the `include` parameter, which are the namespaces you want to apply to the policy to.
+
 The `exclude` parameter specifies the namespaces you explicitly do not want to apply the policy to.
 
 | spec.remediationAction
-| Required. Specifies the remediation of your policy. Enter  `inform`
+| Required.
```
Comment (oafischer) [inline governance/config_policy_ctrl.adoc]: Remove period

File: governance/namespace_policy.adoc [hunk 3]
```diff
@@ -40,47 +40,49 @@ spec:
...
-Set the value to `true` or `false`.
+| Required
+| Set the value to `true` or `false`.
 The `disabled` parameter provides the ability to enable and disable your policies.
 
 | spec.complianceType
-| Required.
-Set the value to `"musthave"`
+| Required
+| Set the value to `"musthave"`
```
Comment (oafischer) [inline governance/namespace_policy.adoc]: Add period.

**Review by oafischer — APPROVED** [review]
Thanks for taking the time to change all the tables @dockerymick! Just see the 2 comments I left.

File: governance/etcd_encryption_policy.adoc (hunk 1 above)
Comment (dhaiducek) [reply governance/etcd_encryption_policy.adoc]: Technically `metadata.namespace` is required and should be singular.

It's the namespace of the policy.

**Pull request merged**